### ISSUE TITLE
Lightbulb for library() for functions from unloaded packages

### ISSUE THREAD

**Issue by int19h** [Microsoft/RTVS#1039]
Currently, if there is a function in your code that is from a package that is not loaded (but installed), the user has to manually discover the package by bringing up function help, seeing which package it is defined it, and adding a `library` statement to load the package.

Since we already scrape all installed packages for Intellisense, we have the list of all functions in all packages. We could proactively match identifiers in editor code against any `library` statements that it contains, and if we notice that some identifier is missing a corresponding `library`, show a lightbulb for that line, and offer to insert a `library` statement at the beginning of the file.

Basically, same as "add missing using" functionality for C#, and "add missing import" for Python.
